sqlalchemy_redshift/dialect.py

```diff
@@ -154,30 +154,31 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
-    "and", "any", "array", "as", "asc", "authorization", "az64",
-    "backup", "between", "binary", "blanksasnull", "both", "bytedict",
-    "bzip2", "case", "cast", "check", "collate", "column", "constraint",
-    "create", "credentials", "cross", "current_date", "current_time",
-    "current_timestamp", "current_user", "current_user_id", "default",
-    "deferrable", "deflate", "defrag", "delta", "delta32k", "desc",
-    "disable", "distinct", "do", "else", "emptyasnull", "enable",
-    "encode", "encrypt", "encryption", "end", "except", "explicit",
-    "false", "for", "foreign", "freeze", "from", "full", "globaldict256",
-    "globaldict64k", "grant", "group", "gzip", "having", "identity",
-    "ignore", "ilike", "in", "initially", "inner", "intersect", "into",
-    "is", "isnull", "join", "language", "leading", "left", "like",
-    "limit", "localtime", "localtimestamp", "lun", "luns", "lzo", "lzop",
-    "minus", "mostly13", "mostly32", "mostly8", "natural", "new", "not",
-    "notnull", "null", "nulls", "off", "offline", "offset", "oid", "old",
-    "on", "only", "open", "or", "order", "outer", "overlaps", "parallel",
-    "partition", "percent", "permissions", "placing", "primary", "raw",
-    "readratio", "recover", "references", "respect", "rejectlog",
-    "resort", "restore", "right", "select", "session_user", "similar",
-    "snapshot", "some", "sysdate", "system", "table", "tag", "tdes",
-    "text255", "text32k", "then", "timestamp", "to", "top", "trailing",
-    "true", "truncatecolumns", "union", "unique", "user", "using",
-    "verbose", "wallet", "when", "where", "with", "without",
+    "aes128", "aes256", "all", "allowoverwrite", "analyse", "analyze", 
+    "and", "any", "array", "as", "asc", "authorization", "az64", 
+    "backup", "between", "binary", "blanksasnull", "both", "bytedict", 
+    "bzip2", "case", "cast", "check", "collate", "column", "constraint", 
+    "create", "credentials", "cross", "current_date", "current_time", 
+    "current_timestamp", "current_user", "current_user_id", "default", 
+    "deferrable", "deflate", "defrag", "delta", "delta32k", "desc", 
+    "disable", "distinct", "do", "else", "emptyasnull", "enable", 
+    "encode", "encrypt", "encryption", "end", "except", "explicit", 
+    "false", "for", "foreign", "freeze", "from", "full", "globaldict256", 
+    "globaldict64k", "grant", "group", "gzip", "having", "identity", 
+    "ignore", "ilike", "in", "initially", "inner", "intersect", "into", 
+    "is", "isnull", "join", "language", "leading", "left", "like", 
+    "limit", "localtime", "localtimestamp", "lun", "luns", "lzo", "lzop", 
+    "minus", "mostly16", "mostly32", "mostly8", "natural", "new", "not", 
+    "notnull", "null", "nulls", "off", "offline", "offset", "oid", "old", 
+    "on", "only", "open", "or", "order", "outer", "overlaps", "parallel", 
+    "partition", "percent", "permissions", "pivot", "placing", "primary", 
```

Comment:
There aren't any differences until this line. Do you know why lines 157 through 173 show up in the diff? Is there some change in whitespace here?

Reply:
I've seen that there were trailing space (due to copy pasting I suppose). I've removed them. 
The diff is cleaner now. :-)